docker-compose.yml

```diff
@@ -0,0 +1,103 @@
+version: '3'
+services:
+  node-0:
+    image: cennznet/cennznet:1.1.1
+    volumes:
+    - ./data/node-0:/mnt/data
+    command:
+      - --chain=/cennznet/genesis/nikau.raw.json
+      - --validator
+      - --alice
+      - --base-path=/mnt/data
+      - --name=alice-${HOSTNAME:-localhost}-0
```

Comment:
Do we need -0, -1 and etc at the end of the names?

Reply:
Yeah it's the docker container names, it could be named something else if we wanted